thentos-core/src/Thentos/Backend/Core.hs

```diff
@@ -120,6 +121,17 @@ actionErrorToServantErr e = do
     _permissions _ = logger DEBUG (ppShow e) >> pure (err401 { errBody = "unauthorized" })
 
 
+-- * custom servers for servant
+
+data Post200 (contentTypes :: [*]) a
+    deriving Typeable
+
+instance ( AllCTRender ctypes a ) => HasServer (Post200 ctypes a) where
+
```

Comment:
(i don't like the above empty line, but there is nothing in the style guide to prevent you from keeping it.)